packages/components/src/popover/types.ts

```diff
@@ -184,6 +190,12 @@ export type PopoverProps = {
 	getAnchorRect?: (
 		fallbackReferenceElement: Element | null
 	) => DomRectWithOwnerDocument;
+	/**
+	 * Used to enable a different visual style for the popover.
+	 *
+	 * @deprecated
+	 */
```

Comment:
Just so this reads better in the Storybook props table (which doesn't show the `@deprecated` tag), let's add a sentence that this is deprecated and what API to use instead.